core/src/main/resources/reference.conf

```diff
@@ -407,6 +406,12 @@ cassandra-query-journal {
     # before the stream starts emitting events.
     offset-scanning-period = 200ms
 
+    # Interval at which events by tag stages trigger a query to look for delayed events from before the
+    # current offset. Delayed events are searched for when a subsequent event for the same persistence id
+    # is received. The purpose of this back track is to find delayed events for persistence ids that are no
+    # longer receiving events. The backtrack is expensive so do not set this too low.
+    back-track-interval = 6s
+
```

Comment:
Would be good to update the docs for eventual-consistency-delay and mention that if it's set too low (or 0s) it will likely result in higher latency since events from different nodes (persistenceIds) will become visible at different points in time. Those delayed events with lower offset will be missed and then picked up by this `back-track-interval` or by chance by next event from same persistenceId.

I think the delay will often be caused by the `flush-interval` on the write side, and flushing from one node may thereby overtake events that are still in the flush buffer on another node. To compensate for that the eventual-consistency-delay should probably not be set lower than flush-interval. Maybe also worth a log warning?

Reply:
I'll do an overhaul of the eventually consistency docs at the end :+1: 

>To compensate for that the eventual-consistency-delay should probably not be set lower than flush-interval. Maybe also worth a log warning?

+1